CHANGELOG.md

```
@@ -12,6 +12,14 @@ _**Note:** Yet to be released breaking changes appear here._
 **Breaking Changes**:
 - The `utils.isNullish` and `utils.isNotNullish` functions are now marked as private.
 They had been made public by mistake, and had been considered internal since their introduction.
+- Some utility functions formerly used to retrieve default value of `CellStateStyle` and `CellStyle` properties, mainly intended to only be used internally, have been removed.
+    - Use the [nullish coalescing operator (??)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)